src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/GeneViewSideBar.tsx

```diff
@@ -32,7 +32,7 @@ type Props = {
 };
 
 const client = new ApolloClient({
-  uri: 'http://hx-rke-wp-webadmin-13-worker-1.caas.ebi.ac.uk:31497'
+  uri: 'http://web-thoas.review.ensembl.org/'
```

Comment:
sneaky 😄

Reply:
Ah! No wonder why my commit was missing from the other branch! 
I will get this reverted in this PR and do this update separetely.